src/main/play-doc/operation/Install.md

```diff
@@ -990,60 +729,189 @@ Proxying application endpoints is required when running more than one instance o
 
 ConductR-HAProxy requires `HAProxy` version 1.5 or newer. This can be satisfied by installing HAProxy natively, such as using a package manager, or by running the HAProxy Docker image. Native HAProxy installation requires less configuration. The Docker image should be used when using CoreOS for cluster nodes.
```

Comment:
Remove 'This can be satisfied by installing HA...' 
Can say 'We'll use ...' in each section as appropriate instead and add cross ref (This requirement can also be met ...) if desired.